pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <artifactId>hugegraph</artifactId>
         <groupId>com.baidu.hugegraph</groupId>
-        <version>0.12.0</version>
+        <version>0.12.1</version>
```

Comment:
there is no hugegraph v0.12.1, will cause not-found-error?

Reply:
ok， I change it to 0.13.0